ui/app/components/secret-edit-toolbar.js

```diff
@@ -12,7 +12,6 @@
  * @secretDataIsAdvanced={{secretDataIsAdvanced}}
  * @showAdvancedMode={{showAdvancedMode}}
  * @modelForData={{this.modelForData}}
- * @navToNearestAncestor={{this.navToNearestAncestor}}
```

Comment:
A mixin method we were not using. We passed it into the secret-delete-menu but didn't actually use it.